src/interpreter/InterpretNot.ts

```diff
@@ -17,15 +17,13 @@ export default function interpretNot(schema: Schema, model: CommonModel, interpr
   if (typeof schema.not === 'object') {
     const notSchema = schema.not;
     const newInterpreterOptions: InterpreterOptions = {
-      ...interpreterOptions, 
-      splitModels: false,
+      ...interpreterOptions,
       allowInheritance: false
     };
-    const interpretedModels = interpreter.interpret(notSchema, newInterpreterOptions);
-    if (interpretedModels.length > 0) {
-      const interpretedModel = interpretedModels[0];
-      if (interpretedModel.type !== undefined) {model.removeType(interpretedModel.type);}
-      if (interpretedModel.enum !== undefined) {model.removeEnum(interpretedModel.enum);}
+    const notModel = interpreter.interpret(notSchema, newInterpreterOptions);
+    if (notModel !== undefined) {
+      if (notModel.type !== undefined) {model.removeType(notModel.type);}
+      if (notModel.enum !== undefined) {model.removeEnum(notModel.enum);}
```

Comment:
I know that `notModel.type !== undefined` is more readable, but maybe we should go with

```js
notModel.type && model.removeType(notModel.type);
```

Reply:
hmmm, good suggestion! I just don't know if I like that format 😄

Reply:
You can do this in the separate PR or create (another) issue 😅

Reply:
This syntax actually violates the eslint rule eslintno-unused-expressions. So it cannot be such a format 🙂

Reply:
So eslint sucks in this place 😄

Reply:
I mean, to me this syntax is honestly better but it is a matter of opinion I guess 😄

Reply:
It was a joke :)

Reply:
😆 Sure! I know you like your sweet short syntax though! So it might as well could have been real xD